docs/development/setup-advanced.md

```diff
@@ -136,6 +137,116 @@ Store.
    environment](../development/setup-vagrant.html#step-4-developing),
    ignoring the parts about `vagrant` (since you're not using it).
 
+## Using the Vagrant Hyper-V provider
```

Comment:
Oh, one more small comment: this heading should mention Windows. Maybe add " (on Windows)" to the end.

(Someone who knows what Hyper-V is will know it means Windows, of course, but others may not. This helps people on a given operating system quickly scan to see the options that might apply to them.)